Example/Tests/PageStateViewTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PageStateViewTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 4/13/18.
+//  Copyright © 2018 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class PageStateViewTests: XCTestCase {
+    
+    var statusView: PageStateView!
+    
+    override func setUp() {
+        super.setUp()
+        statusView = PageStateView(frame: .zero)
+    }
+    
+    func testLoadingState() {
+        statusView.update(to: .loading)
+        XCTAssertNil(statusView.icon.image, "icon image should be nil when it is loading")
```

Comment:
I'm not sure that this tests. Just setting the state doesn't mean the image is shown or even set. And as far as I saw, it's just an empty image view at that point. So I'd argue that this assertion is not really useful.

Same thing applies to the following two methods.

Reply:
Makes sense for the first one but yeah, for the next two tests we should check the image itself